samples/IISSample/Startup.cs

```diff
@@ -15,13 +17,14 @@ public void ConfigureServices(IServiceCollection services)
             // These two middleware are registered via an IStartupFilter in UseIISIntegration but you can configure them here.
             services.Configure<IISOptions>(options =>
             {
+                options.AuthenticationDisplayName = "Windows Auth";
```

Comment:
Should just default this to the scheme like everything else to be consistent: IISDefaults.AuthenticationScheme.  We don't suffix anything else with Auth today